conf/pawsey_setonix.config

```diff
@@ -0,0 +1,29 @@
+// Pawsey Setonix nf-core configuration profile
+params {
+  config_profile_description = 'Pawsey Setonix HPC profile provided by nf-core/configs'
+  config_profile_contact = 'Sarah Beecroft (@SarahBeecroft), Georgie Samaha (@georgiesamaha)'
+  config_profile_url = 'https://support.pawsey.org.au/documentation/display/US/Setonix+User+Guide'
+}
+
+// Enable use of Singularity to run containers
+singularity {
+  enabled = true
+  autoMounts = true
+  autoCleanUp = true
```

Comment:
Ooh that's a new one for me, what does that do? Or is it from edge version

Reply:
Was a suggestion from Pawsey colleague, struggling to find documentation on it though... Will follow up